src/redux/Graphql/reducer.js

```diff
@@ -10,9 +10,15 @@ export default function graphqlReducer(state = initState, action) {
     case actions.GET_TOPICS_RETURN: {
       return state.set('getTopicsReturn', action.value);
     }
+    case actions.GET_MORE_TOPICS_RETURN: {
```

Comment:
remove, use the same reducer `GET_TOPICS_RETURN`

Reply:
If using the same reducer, how can i append the new data to previous one? Since sometimes it need set totally new. sometimes append.